pages.pt_BR/freebsd/base64.md

```diff
@@ -0,0 +1,28 @@
+# base64
+
+> Codifica ou decodifica arquivo ou `stdin` de/para base64, para `stdout` ou outro arquivo.
+> Mais informações: <https://man.freebsd.org/cgi/man.cgi?query=base64>.
+
+- Codifica um arquivo para `stdout`:
+
+`base64 {{-i|--input}} {{caminho/para/arquivo}}`
+
+- Codifica um arquivo para o arquivo de saída especificado:
+
+`base64 {{-i|--input}} {{caminho/para/arquivo_de_entrada}} {{-o|--output}} {{caminho/para/arquivo_de_saída}}`
+
+- Enrola saída codificada a uma largura específica (‘0’ desabilita encapsulamento):
```

Comment:
```suggestion
- Quebra (insere uma quebra de linha) a saída codificada em uma largura específica (‘0’ desabilita encapsulamento):
```
I know this is not the literal translation, but it makes it more understandable than "enrola".